### ISSUE TITLE
How to report a package build failure [explanation video]

### ISSUE THREAD

**Issue by autoantwort** [microsoft/vcpkg#30604]
https://user-images.githubusercontent.com/41973254/229313721-b82c2d3d-b0f9-4e31-98df-08d75831f695.mp4

<br>

This video assumes that you are using the latest vcpkg version. If you don't use the latest version or are not sure:
1. `git pull`
2. `./bootstrap-vcpkg.sh`(unix) `./bootstrap-vcpkg.bat`(windows)

**Comment by autoantwort** [comment]
Maybe pin this issue so that it is more visible.

**Comment by Neumann-A** [comment]
My opinion:
I find this good but I have the following problems with it:
- way too fast
- it took me several repeated views to even understand what is going on (and why you are doing it)

Having a spoken description alongside the video probably would slow down the pace a lot and make it easier to follow.

**Comment by Abdalslam-Alromimah** [comment]
./vcpkg install boost:x86-windows
 
 the error:
 
 CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: E:/src/vcpkg/downloads/tools/cmake-3.25.1-windows/cmake-3.25.1-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j9
    Working Directory: E:/src/vcpkg/buildtrees/boost-serialization/x86-windows-dbg
    See logs for more information:
      E:\src\vcpkg\buildtrees\boost-serialization\install-x86-windows-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  installed/x64-windows/share/boost-build/boost-modular-build.cmake:140 (vcpkg_cmake_install)
  ports/boost-serialization/portfile.cmake:12 (boost_modular_build)
  scripts/ports.cmake:147 (include)


error: building boost-serialization:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+boost-serialization
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?title=[boost-serialization]+Build+error&body=Copy+issue+body+from+E%3A%2Fsrc%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md

**Comment by FrankXie05** [comment]
@Abdalslam-Alromimah Please submit a new issue, this issue guides how to submit an issue. :)

**Comment by Guest-615695028** [comment]
#33680 Despite using the latest vcpkg version I still failed to build `vcpkg install glfw3 glad glm`.

**Comment by pedro-w** [comment]
Just like @Neumann-A I found the video hard to follow. As far as I can tell, the instruction is:
* When a build failure occurs there will be some red error text which includes "You can submit a new issue at:" followed by a long URL.
* Copy *all* of the URL and paste it into the location bar in your browser (just clicking on the link won't work)
* A new Issue will appear, with a file named in the content section
* Find that file and open it in an editor, select all, copy and and paste it into the new Issue, replacing whatever is in the content section
